README.md

```diff
@@ -37,3 +37,15 @@ The relayer consists of the following components:
 <div align="center">
   <img src="resources/relayer-diagram.png?raw=true">
 </div>
+
+## Testing
+
+---
```

Comment:
Nit: The horizontal line will be included below the headings automatically. It looks like we've doubled up in this case. Could we remove the "---" from the README here and above?